.editor_settings

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
This file not related to the issue, so you can remove it from the PR (by remove & commit in your repository)